spec/system/oh_audio_front_end_spec.rb

```diff
@@ -59,6 +67,24 @@
 
       click_on "Downloads"
 
+
+      # In oh_audio_work_show_decorator.rb we specify that only PDFs should be linked.
```

Comment:
Added an extra test here for untested functionality in `oh_audio_work_show_decorator.rb` -- out of scope, I know, but seemed like a good idea at the time.